src/avt/Expressions/Abstract/avtExpressionDataTreeIterator.C

```diff
@@ -36,11 +36,20 @@
 //  Programmer: Hank Childs
 //  Creation:   December 27, 2004
 //
+//  Modifications:
+//
+//    Eddie Rusu, Mon Dec 30 20:41:07 PST 2019
+//    Initialize the volumeDependent array. Default dependency to false.
+//
 // ****************************************************************************
 
 avtExpressionDataTreeIterator::avtExpressionDataTreeIterator()
 {
-    ;
+    volumeDependent = vtkBitArray::New();
+    volumeDependent->SetName("VolumeDependent");
+    volumeDependent->SetNumberOfComponents(1);
+    volumeDependent->SetNumberOfTuples(1);
+    volumeDependent->SetComponent(0, 0, false); // Default volume dependency to false
```

Comment:
I believe this will lead to a memory leak. You should add a volumeDependent->Delete() to the destructor.